addons/spreadsheet/static/src/@types/global_filter.d.ts

```diff
@@ -3,9 +3,13 @@ import { DomainListRepr } from "@web/core/domain";
 
 declare module "@spreadsheet" {
```

Comment:
Date format isn't the same in the dropdown and in the input/searchbar. Should always be the user date format

![image](https://github.com/user-attachments/assets/5569fafa-3e24-4051-86e2-4d87946fe0c1)

Reply:
Fixed in https://github.com/odoo/odoo/pull/214344/files#diff-714723b6e25d85e13ce423b00d948109278f856f92d261dbd1d71b5fe60cd829R55-R68